.mvn/extensions.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<extensions>
```

Comment:
Any reason not to include the schema stuf?

```
<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.1.0 https://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
```

Reply:
Nope, happy to add

Reply:
resolved in https://github.com/kroxylicious/kroxylicious/pull/2002/commits/31db43dc00d8f30c6ccaf4c62ae09f940eab7f38